api/reference/providers.adoc

```diff
@@ -226,7 +227,7 @@ Compound credentials set:
 [[specifying-connection-configurations]]
 === Specifying Connection Configurations
 When creating or updating providers, connection configurations can be set. Connection configurations can be used
-to specify resources such as an amqp event provider for Openstack or for adding Hawkular metrics.
+to specify resources such as an amqp event provider for Openstack, adding Hawkular metrics, or an Authentication token.
```

Comment:
Sorry for the nitpicks - can we make "authentication token" lower-case, and I would suggest marking up "ampq" with ` symbols so it doesn't get translated accidentally and stands out more. Thanks!